platforms/apple/Sources/Nimbus/Callback.swift

```
@@ -29,22 +29,36 @@ class Callback: Callable {
     }
 
     func call(args: [Any]) throws -> Any {
-        var jsonString: String = "[]"
-        if let encodables = args as? [Encodable] {
-            let jsonEncoder = JSONEncoder()
-            let jsonData = try jsonEncoder.encode(EncodableValue.array(encodables))
-            jsonString = String(data: jsonData, encoding: .utf8)!
-        } else {
-            // Parameters passed to callback are implied that they
-            // conform to Encodable protocol.  If for some reason
-            // any elements don't throw parameter error.
-            throw ParameterError()
+        let jsonEncoder = JSONEncoder()
+        let jsonArgs = try args.map { arg -> String in
+            if let encodable = arg as? Encodable {
+                let jsonData = try jsonEncoder.encode(EncodableValue.value(encodable))
+                let jsonString = String(data: jsonData, encoding: .utf8)!
+                return jsonString
+            } else if arg is NSArray || arg is NSDictionary {
+                // swiftlint:disable:next force_try
+                let data = try! JSONSerialization.data(withJSONObject: arg, options: [])
```

Just use `try` here, without the forcing. This method is `throws` so it's okay to let the error bubble up out of this method. Ideally we want to not add any additional force trys and get rid of the few existing cases we do have.